app/src/migration/java/org/mozilla/fenix/MigrationDecisionActivity.kt

```diff
@@ -39,6 +39,6 @@ class MigrationDecisionActivity : Activity() {
         // and then we switch to the actual activity without an animation. This visually looks like
         // a faster start than launching this activity invisibly and switching to the actual
         // activity after that.
-        overridePendingTransition(0, 0)
+        overridePendingTransition(0, R.anim.placeholder_animation)
```

Comment:
Can you elaborate on the comment on why having a placeholder is helpful?

Reply:
Having the placeholder animation seems to force the app to create an animation where the `0` just shuts it down before the other activity starts. This was more of me playing around with the animation and finding a place holder. This is more a conclusion I came to from playing around with the different animations